### PR TITLE
IM-DnD

### PR DESCRIPTION
**Pull request by Perfoys**
Created functionality for Drag and Drop between columns
Created new component DropWrapper
Modified sprintReducer

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+import Timer from '../common/Timer';
 
 const Header = ({ sprint }) => {
   return (
     <header className={style.header}>
       <h1 className={style.title}>{ sprint.title }</h1>
       <div>{ sprint.startDate.toDateString() }</div>
       <div>{ sprint.endDate.toDateString() }</div>
-      <Timer endDate={sprint.endDate} />
+      <div>Days left: <Timer endDate={sprint.endDate} /></div>
```
Comment (Smolyaninov-Dmitriy) [inline src/components/header/Header.js:11]: in my opinion, the previous implementation was better

File: src/components/table/Column.js [hunk 2]
```diff
@@ -1,20 +1,53 @@
...
+  const onDrop = (item, monitor, title) => {
+    if (item.status === title) return;
+    dispatch(changeTaskStatus({ item: item, title: title }));
+    setCards(prevCards => update(prevCards, {
+      $push: [item]
+    }));
+  };
+
+  useEffect(() => {
+    setCards(tasks);
```
Comment (Smolyaninov-Dmitriy) [inline src/components/table/Column.js:33]: why u don't use this property instead 'cards'?

File: src/components/table/MembersTable.js [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
+const MembersTable = ({ tasks }) => {
   return (
     <div className={style.table}>
-      <Column title="To do" tasks={toDoTasks} />
-      <Column title="In Progress" tasks={inProgressTasks} />
-      <Column title="In Review" tasks={inReviewTasks} />
-      <Column title="Done" tasks={doneTasks} />
+      <Column
+        title={status.TODO}
+        tasks={tasks.filter(item => item && item.status === status.TODO)}
```
[the remark this answers is not in the document]
Comment (Perfoys) [reply src/components/table/MembersTable.js:12]: It's a good idea, I created filterTasks function. Thanks

File: src/components/table/Column.js [hunk 4]
```diff
@@ -1,14 +1,44 @@
...
+import { useDispatch } from 'react-redux';
+import { changeTaskStatus } from '../../reducers/sprintReducer';
 import TaskCard from './TaskCard';
 import style from './column.module.scss';
+import DropWrapper from './DropWrapper';
 
 const Column = ({ title, tasks }) => {
+  const [cards, setCards] = useState(tasks);
+  const dispatch = useDispatch();
+  const moveCard = (dragIndex, hoverIndex) => {
```
[the remark this answers is not in the document]
Comment (Perfoys) [reply src/components/table/Column.js]: Yes, I think we should, thank you. Solved it

File: src/components/table/Column.js (hunk 2 above)
Comment (Perfoys) [reply src/components/table/Column.js]: It was a bad idea, thanks for the comment. It helped me a lot

File: src/components/header/Header.js (hunk 1 above)
Comment (Perfoys) [reply src/components/header/Header.js]: I tried to return this to previous version with modifications

**Pull request merged**